src/main/java/liquibase/ext/spanner/snapshotgenerator/SequenceSnapshotGeneratorSpanner.java

```diff
@@ -0,0 +1,74 @@
+package liquibase.ext.spanner.snapshotgenerator;
```

Comment:
nit: add a copyright header. Just copy-paste one from one of the other files (and update the copyright year to 2024)

Reply:
added